packages/client/components/JiraScopingSearchResultItem.tsx

```diff
@@ -52,11 +52,11 @@ const JiraScopingSearchResultItem = (props: Props) => {
   const {id: serviceTaskId, issueKey, summary, url} = issue
   const isSelected = usedServiceTaskIds.has(serviceTaskId)
   const atmosphere = useAtmosphere()
-  const {onCompleted, onError, submitMutation, submitting} = useMutationProps()
+  const {onCompleted, onError, submitMutation} = useMutationProps()
   const disabled = !isSelected && usedServiceTaskIds.size >= Threshold.MAX_POKER_STORIES
   const isTemp = isTempId(serviceTaskId)
   const onClick = () => {
-    if (submitting || disabled || isTemp) return
+    if (disabled || isTemp) return
```

Comment:
Should we do the same for `JiraScopingSelectAllIssues`? The only potential problem I see, this mutation can be really slow so it potentially keeps updating the UI after you already finished clicking. On the other hand, not allowing users to click at all while mutation is in progress is really bad UX. 

I'd personally do the same, and if performance problems starts to annoy us, create a separate task to fix that

Reply:
i'll make the change